hugr-py/tests/test_hugr_build.py

```diff
@@ -0,0 +1,176 @@
+import subprocess
+
+from hugr.hugr import Dfg, Hugr, DummyOp, Node
+import hugr.serialization.tys as stys
+import hugr.serialization.ops as sops
+import pytest
+
+BOOL_T = stys.Type(stys.SumType(stys.UnitSum(size=2)))
+QB_T = stys.Type(stys.Qubit())
+ARG_5 = stys.TypeArg(stys.BoundedNatArg(n=5))
+INT_T = stys.Type(
+    stys.Opaque(
+        extension="arithmetic.int.types",
+        id="int",
+        args=[ARG_5],
+        bound=stys.TypeBound.Eq,
+    )
+)
+
+NOT_OP = DummyOp(
+    # TODO get from YAML
+    sops.CustomOp(
+        parent=-1,
```

Comment:
Would probably friendlier for users not to need to say `parent = -1`; is it avoidable?